packages/react-core/src/modules/slack/context/KnockSlackProvider.tsx

```diff
@@ -0,0 +1,83 @@
+import { useSlackConnectionStatus } from "..";
+import Knock from "@knocklabs/client";
+import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
+import * as React from "react";
+
+import { slackProviderKey } from "../../core";
+import { useKnockClient } from "../../core";
+import { ConnectionStatus } from "../hooks/useSlackConnectionStatus";
+
+const queryClient = new QueryClient();
+
+export interface KnockSlackProviderState {
+  knock: Knock;
```

Comment:
I don't think we should reexpose the knock instance, better to just have anything that needs it call `useKnockClient`

Reply:
ok cool